plugins/topology/src/components/Graph/TopologyComponentFactory.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import {
+  GraphElement,
+  ComponentFactory,
+  withDragNode,
+  withSelection,
+  ModelKind,
+  withPanZoom,
+} from '@patternfly/react-topology';
+import GroupNode from './GroupNode';
+import EdgeConnect from './EdgeConnect';
+import WorkloadNode from './WorkloadNode';
+import { TYPE_APPLICATION_GROUP, TYPE_CONNECTS_TO } from '../../const';
+import DefaultGraph from './DefaultGraph';
+
+const TopologyComponentFactory: ComponentFactory = (
+  kind: ModelKind,
+  type: string,
+): React.ComponentType<{ element: GraphElement }> | undefined => {
+  if (kind === ModelKind.graph) {
+    return withPanZoom()(DefaultGraph);
+  }
+  switch (type) {
+    case 'workload':
```

Comment:
This should be a const as well (it't used in [`data-transforms/data-transformer.ts`](https://github.com/janus-idp/backstage-plugins/pull/104/files#diff-0253d70e4353e47b4ecae9126d55b0479178517ec598e1861b889c9dd04cd53fR19) as well)